src/App.js

```diff
@@ -92,7 +85,7 @@ class App extends React.Component {
 
       this.setState({
         result: finalResult + "",
-        lastFormula: nextEquation,
+        lastEquation: nextEquation,
```

Comment:
Here's an example of something that may have been easier to catch if adding and using lastEquation were in a single commit.

Reply:
I'm looking in the commit titled "repeat operation every time equals is pressed"

Reply:
Ahhhh....YES! Great point. I spent so much time on a bug only to discover this as the problem/